packages/components/src/draggable/style.scss

```diff
@@ -23,3 +23,8 @@ body.is-dragging-components-draggable {
 		display: none;
 	}
 }
+
+// This is only for testing.
+.is-dragging.is-selected {
+	display: none !important;
```

Comment:
did you try with `opacity: .5` instead here? Would work much better with for example reordering columns.

Reply:
We tried with ´opacity: 0.7`. 

As I see it, there are two paths forward:

1. Go with the more radical solution of **pulling the block out of the flow**, literally lifting it off the page. This benefits very big blocks which take up a lot of vertical space, but can cause jumpiness.
2. Go with the opacity change, and the chip, which is still an improvement over what we're shipping, but does not have the space saving benefits.

Honestly I think we should get option 1 in good shape, try it in the plugin, and then go with option 2 if we find after using it that the jump is jarring.